### PR TITLE
Payables additional changes/refactoring

### PR DESCRIPTION
**Pull request by aninaslyan**
- Payables additional changes
- Receivables refactoring

### CODE REVIEW

**Comment by igorstdev** [comment]
@aninaslyan jan did you notes the conflict?

File: src/src_V2/modules/manager/financial/payables/control-panel/ControlPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        />
+      </Box>
+
+      <Box className={styles.filterDiv}>
+        <Button variant='outlined' startIcon={<FilterAlt />}>
+          Filter
+        </Button>
+      </Box>
+
+      {/*<Box className={styles.optionsDiv}>*/}
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/payables/control-panel/ControlPanel.jsx:24]: @aninaslyan jan need we this commented section?

File: src/src_V2/modules/manager/financial/payables/edit-drawer/EditDrawer.jsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    }
+  };
+
+  const handleSaveChanges = () => {
+    // first - backend save request, then...
+    console.log('saved');
+    closePopupsAndDrawer();
+  };
+
+  useEffect(() => toggleDrawer(isOpened), [isOpened]);
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/payables/edit-drawer/EditDrawer.jsx]: @aninaslyan jan need we pass isOpened argument here? I think it's component level, hm?

File: src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx [hunk 3]
```diff
@@ -165,11 +174,47 @@ export default function PaymentHistory({
...
+            `Supported file types are - ${acceptedFileTypes.join(', ')}`
+          );
+        } else {
+          const modifiedHistory = paymentHistory.map((history) => {
+            if (history.id === historyIndex) {
+              history.attachment = newFile;
+            }
+            return history;
+          });
+          setFieldValue('paymentHistory', [...modifiedHistory]);
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx]: why we need to construct another array here?

File: src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx [hunk 4]
```diff
@@ -199,10 +244,10 @@ export default function PaymentHistory({
           <Table className={styles.historyTable}>
             <TableHead>
               <TableRow>
-                {columnKeys.map((c) => {
+                {columnKeys.map((c, index) => {
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx]: it's minor thing but i think we can change `c` argument here, at lest column or item. Hm?

File: src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx [hunk 5]
```diff
@@ -219,37 +264,46 @@ export default function PaymentHistory({
...
+                      ref={hiddenFileInput}
+                      onChange={handleAddAttachment}
+                      accept='image/*, .pdf'
+                    />
+                  </TableCell>
+
                   {/*Delete Row*/}
-                  <TableCell>
-                    <IconButton onClick={() => handleDeleteHistory(i)}>
+                  <TableCell key={`delete_${row.id}`}>
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/payables/edit-drawer/payment-history/PaymentHistory.jsx]: need we key prop here?

File: src/src_V2/modules/manager/financial/receivables/control-panel/ControlPanel.jsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        />
+      </Box>
+
+      <Box className={styles.filterDiv}>
+        <Button variant='outlined' startIcon={<FilterAlt />}>
+          Filter
+        </Button>
+      </Box>
+
+      {/*<Box className={styles.optionsDiv}>*/}
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/financial/receivables/control-panel/ControlPanel.jsx:24]: The same comment here(for commented code)?

File: src/src_V2/modules/manager/supplier-management/accommodation/list/MultipleSelectCheckmarks.jsx [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
-import MenuItem from '@mui/material/MenuItem';
+import Checkbox from '@mui/material/Checkbox';
 import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
 import ListItemText from '@mui/material/ListItemText';
+import MenuItem from '@mui/material/MenuItem';
+import OutlinedInput from '@mui/material/OutlinedInput';
 import Select from '@mui/material/Select';
-import Checkbox from '@mui/material/Checkbox';
+import * as React from 'react';
```
Comment (igortokhunts) [inline src/src_V2/modules/manager/supplier-management/accommodation/list/MultipleSelectCheckmarks.jsx]: please remove this import

**Review by igortokhunts — CHANGES_REQUESTED** [review]

File: src/src_V2/modules/manager/financial/payables/control-panel/ControlPanel.jsx (hunk 1 above)
Comment (aninaslyan) [reply src/src_V2/modules/manager/financial/payables/control-panel/ControlPanel.jsx:24]: Maybe we'll manage or in the future, someone else manages to implement the functionality of the button. The button also has corresponding styles. That's why I left it here.

File: src/src_V2/modules/manager/supplier-management/accommodation/list/MultipleSelectCheckmarks.jsx (hunk 7 above)
Comment (aninaslyan) [reply src/src_V2/modules/manager/supplier-management/accommodation/list/MultipleSelectCheckmarks.jsx]: import * as React from 'react'; was previously there. This is the change of `importOrderSortSpecifiers`. Will change it accordingly.

**Review by igortokhunts — APPROVED** [review]

**Comment by igortokhunts** [comment]
@aninaslyan jan the conflict came from other PRs merging, please take a look.

**Pull request merged**